otm-bridge.html

```diff
@@ -191,7 +210,7 @@ <h3><dfn>Deposit Content</dfn></h3>
         <p>Requests that a set of content be deposited into the DDP</p>
         <ul>
           <li>Request Type: <code>POST</code></li>
-          <li>Request Path: <code>/deposit ? deposit-format=</code></li>
+          <li>Request Path: <code>/deposit ? deposit-format</code></li>

```

Comment:
@no-reply I doubt this change (and the similar ones below it) was intentional. In any case, they will need to be reverted before merge.